components/setup/BuildInstruction.tsx

```diff
@@ -69,18 +91,18 @@ const BuildInstruction: React.FC<BuildInstructionProps> = ({ selectedHosting, us
 			});
 	}, [userId])
 
+	if (!installId) {
+		return (<div className="flex items-center gap-4">
+			<p>Something went wrong while fetching install id.</p>
+		</div>);
+	}
 	if (selectedHosting === 'selfhosting') {
 		if (isGetInstallIdLoading) {
 			return (<>
 				<div className='inline-block border-4 border-t-primary-main rounded-full w-6 h-6 animate-spin mx-2'></div>
 				Generating topic name...
 			</>);
 		}
-		if (!installId) {
-			return (<div className="flex items-center gap-4">
-				<p>Something went wrong while fetching install id.</p>
-			</div>);
-		}
```

Comment:
Why did you remove this? This will always be required, right?

Reply:
I shifted this outside the function.